src/styles/components/_navbar.scss

```diff
@@ -48,16 +48,48 @@
     text-decoration: none;
 
     &--sign-in {
-      // TODO: for future implementation
+      color: $dark;
+
+      @include respond-below('sm') {
+        display: none;
+      }
+
+      @include respond-below('md') {
+        font-size: rem(12px);
+      }
     }
 
     &--sign-up {
-      // TODO: for future implementation
+      padding: rem(6px 15px);
+
+      border-radius: rem(26px);
+      border: solid rem(3px) $light;
+      box-shadow: rem(0 3px 6px 0) rgba(0, 0, 0, 0.16);
+
+      color: $light;
+      background-image: linear-gradient(
+        111deg,
+        $cornflower-blue 1%,
+        $royal-blue 101%
+      );
+
+      @include button-darken-effect($cornflower-blue);
+
+      @include respond-below('sm') {
+        display: none;
+      }
+
+      @include respond-below('md') {
+        padding: rem(2px 10px);
+
+        font-size: rem(12px);
+      }
     }
```

Comment:
I don't think using `respond-below` is a good practice for [mobile first](https://medium.com/@Vincentxia77/what-is-mobile-first-design-why-its-important-how-to-make-it-7d3cf2e29d00) approach.